scripts/ci/pre_commit/compile_www_assets.py

```diff
@@ -71,4 +72,20 @@ def get_directory_hash(directory: Path, skip_path_regexp: str | None = None) ->
     subprocess.check_call(["yarn", "install", "--frozen-lockfile"], cwd=os.fspath(www_directory))
     subprocess.check_call(["yarn", "run", "build"], cwd=os.fspath(www_directory), env=env)
     new_hash = get_directory_hash(www_directory, skip_path_regexp=r".*node_modules.*")
-    WWW_HASH_FILE.write_text(new_hash)
+    www_hash_file.write_text(new_hash)
+
+
+def is_fab_provider_installed() -> bool:
+    return importlib.util.find_spec("airflow.providers.fab") is not None
```

Comment:
Ups, this change in this function broke main :-( 
https://github.com/apache/airflow/actions/runs/12399970802/job/34616478712

I assume we need to find another method to check if FAB is installed.

Reply:
Propose to fix via #45058